src/helpers/auth.js

```diff
@@ -1,18 +1,65 @@
 import AuthApi from 'adapters/api/auth';
+import UserApi from 'adapters/api/user';
+
+import sessionManager from 'helpers/sessionManager';
+import tokenManager from 'helpers/tokenManager';
 
 const Auth = () => {
   const login = async (email, password) => {
```

Comment:
I disagree with specific actions should be tied to screens. These actions are core business logic - so not helpers - so should not be coupled with UI. 

As for the usage of `requestInterceptors` of Axios, I agree with Rafayet. This would keep all request management logic in one place (including retries and token refresh).